assets/aws/files/install.sh

```diff
@@ -78,7 +78,7 @@ else
             mv ${TARBALL_FILENAME} /tmp/teleport.tar.gz
         else
             echo "Installing OSS Teleport version ${TELEPORT_VERSION}"
-            curl ${CURL_OPTS} -o teleport.tar.gz https://get.gravitational.com/teleport/${TELEPORT_VERSION}/teleport-v${TELEPORT_VERSION}-linux-amd64-bin.tar.gz
+            curl ${CURL_OPTS} -o teleport.tar.gz https://asset.idemeumlab.com/teleport/${TELEPORT_VERSION}/teleport-v${TELEPORT_VERSION}-linux-amd64-bin.tar.gz
```

Comment:
Use a property to control the asset CDN
it will be different for staging vs production
cdnUrl -> asset.idemeum.com on production
cdnUrl -> asset.idemeumlab.com on staging

Reply:
We will only upload that to staging and then create AMI and share with all accounts.
I will be updating notion with steps to build and prep for automation.